core-data/resolvers.js

```diff
@@ -29,8 +29,8 @@ export async function* getCategories() {
  * Requests authors from the REST API.
  */
 export async function* getAuthors() {
-	const authors = await apiRequest( { path: '/wp/v2/users/?who=authors' } );
-	yield receiveAuthors( authors );
+	const users = await apiRequest( { path: '/wp/v2/users/?who=authors' } );
+	yield receiveUserQuery( 'authors', users );
```

Comment:
I'm hard-coding the queryID ('authors') for now but it could be generated from the query args if we want to generalize this later.